components/popover/index.js

```diff
@@ -271,7 +293,7 @@ class Popover extends Component {
 
 		// Apply focus return behavior except when default focus on open
 		// behavior is disabled.
-		if ( false !== focusOnMount ) {
+		if ( ! focusOnMount ) {
 			content = <FocusManaged>{ content }</FocusManaged>;
 		}
```

Comment:
I'm not sure I understand this change. Unless I'm missing something, the "focus return" behavior should be managed if `focusOnMount` is truthy. /Cc @tofumatt @noisysocks

Reply:
Oh shoot, I think I had a brain fail. It sucks that tests didn't catch this but I read this as `false === focusOnMount` not `false !== focusOnMount`… because the latter seems a weird way to write it.

Follow-up/fix PR incoming, my bad.

Reply:
I think the previous syntax allowed teh prop to be omitted (`null`) and still have focus managed by default?

Reply:
It has a default prop now; omitting it doesn't set it to `null`, it sets it to `"firstElement"`:

https://github.com/WordPress/gutenberg/blob/master/components/popover/index.js#L315

Reply:
Yeah, sorry for cross-commenting on the new PR 😉